diff --git a/packages/tldraw/src/lib/shapes/geo/getGeoShapePath.ts b/packages/tldraw/src/lib/shapes/geo/getGeoShapePath.ts
index da8719682..a6076ecd9 100644
--- a/packages/tldraw/src/lib/shapes/geo/getGeoShapePath.ts
+++ b/packages/tldraw/src/lib/shapes/geo/getGeoShapePath.ts
@@ -326,11 +326,11 @@ export const defaultGeoTypeDefinitions = {
 			const o = w / 4
 			const k = h / 4
 			return new PathBuilder()
-				.moveTo(cx, h, { geometry: { isFilled } })
-				.cubicBezierTo(0, k * 1.2, o * 1.5, k * 3, 0, k * 2.5)
-				.cubicBezierTo(cx, k * 0.9, 0, -k * 0.32, o * 1.85, -k * 0.32)
-				.cubicBezierTo(w, k * 1.2, o * 2.15, -k * 0.32, w, -k * 0.32)
-				.cubicBezierTo(cx, h, w, k * 2.5, o * 2.5, k * 3)
+				.moveTo(cx, 0, { geometry: { isFilled } })
+				.cubicBezierTo(0, h - k * 1.2, o * 1.5, h - k * 3, 0, h - k * 2.5)
+				.cubicBezierTo(cx, h - k * 0.9, 0, h + k * 0.32, o * 1.85, h + k * 0.32)
+				.cubicBezierTo(w, h - k * 1.2, o * 2.15, h + k * 0.32, w, h + k * 0.32)
+				.cubicBezierTo(cx, 0, w, h - k * 2.5, o * 2.5, h - k * 3)
 				.close()
 		},
 	},
